content/basics/index.md

```diff
@@ -131,14 +129,14 @@ in the program.
 The `let` prefix is slightly more flexible in that it can be reassigned, and should be used when `const` cannot.
 
 The use of `var` no longer makes sense in modern JavaScript; it's like `let` in that it can be reassigned,
-but is also 'hoisted' to the top of the current code block.
+but is also ['hoisted'](https://developer.mozilla.org/en-US/docs/Glossary/Hoisting) to the top of the current function scope.
 
 Given this is a common source of bugs, and given it offers no real advantage over `let`, you should really always use `let`.
 
 ```js
-const myVar      = 10;
-let   myOtherVar = 'Dog';
-var   myVarVar   = []; // should be let or const
+const myVar = 'Dog';
+let myOtherVar = 'Dog';
+var myVarVar = 'Dog'; // should be let or const
```

Comment:
Keeping these values all the same to make it clear the thing we are talking about here is the variable declaration type